presidio-anonymizer/presidio_anonymizer/services/validators.py

```diff
@@ -20,8 +20,22 @@ def validate_parameter_in_range(
         )
 
 
+def validate_parameter_exists(parameter_value, entity: str, parameter_name: str):
+    """Validate parameter exists and not only empty."""
+    if not parameter_value:
+        raise InvalidParamException(
+            f"Invalid input, {entity} must contain {parameter_name}")
+
+
+def validate_parameter_not_empty(parameter_value, entity: str, parameter_name: str):
+    """Validate parameter is not empty."""
+    if parameter_value is None:
+        raise InvalidParamException(
+            f"Invalid input, {entity} must contain {parameter_name}")
+
+
```

Comment:
I think both can be unified? 
Also we have an empty check below in validate_parameter, you just wanted a different error message? We can also change the error there if you feel it's not descriptive enough

Reply:
the condition is different.

Reply:
it's not the same condition or same result.